modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/js/EmptyResultMessagePropsTransformer.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {openSimpleInputModal} from 'frontend-js-web';
+
+function addAssetListEntry(itemData, namespace) {
+	openSimpleInputModal({
+		dialogTitle: itemData.title,
+		formSubmitURL: itemData.addAssetListEntryURL,
+		mainFieldLabel: Liferay.Language.get('title'),
+		mainFieldName: 'title',
+		mainFieldPlaceholder: Liferay.Language.get('title'),
+		namespace,
+	});
+}
+
+export default function propsTrasformer({items, portletNamespace, ...props}) {
```

Comment:
Typo, should be propsTra**n**sformer

Reply:
Also, I see in other `propsTransformer`s that we are inconsistent about using `...props` and `...otherProps`. I think in this context `otherProps` makes more sense, `props` should represent the complete props object. Clay is also consistent in using `otherProps` in this way.

Reply:
Sorry for the typo and thanks for the catch.

If you think it's better to call `props` ... `otherProps`, I'll change it - but I don't think it it's that important and I also don't agree with the reasoning.

Reply:
Fixed [here](https://github.com/liferay-frontend/liferay-portal/blob/5671232343a89db53632a5824a66b25758901a9b/modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/js/EmptyResultMessagePropsTransformer.js#L31)